add tabs on main page:

aloescript opened the pull request:
- refactor desktop and mobile layout
- fill tabs with components

aloescript committed:
add tabs on main page:

- refactor desktop and mobile layout
- fill tabs with components

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on 03-lection1/01-button/button.css:
```diff
@@ -21,6 +21,11 @@
   pointer-events: none;
 }
 
+.button-text {
```
что это за другой блок внутри блока .button?)
или это модификатор .button_text?

GSemikozov commented on 07-lection5/03-toggler/index.html:
```diff
@@ -1,31 +1,34 @@
+</head>
+
+<body>
+  <div class="toggler">
```
в случае с инпутами как раз все оставляем завернутым в label - у нас же таким образом привязка к изменению состояния инпута + не надо задавать id

GSemikozov commented on components/tabs/tabs.css:
```diff
@@ -0,0 +1,47 @@
+  border-bottom: 5px solid var(--black);
+}
+
+.tabs__scroller-item_active {
```
чуть выше уже есть такой класс

aloescript committed:
Merge branch 'master' into tabs

aloescript committed:
fix review items:

amend button text modificator
wrap toggler input with label
remove class duplication in tabs

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on components/card/card.css:
```diff
@@ -27,10 +28,32 @@
+
+@media screen and (max-width: 767px) {
+  .card {
+    width: 80%;
```
лишнее, у нас контейнер добавит отступов по краям тобы карточка не прилипала к краям
